### PR TITLE
Add none model choice and fix bulk import alert

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/pages/bulkImport/BulkImport.jsx [hunk 1]
```diff
@@ -23,10 +23,9 @@ export default function BulkImport() {
   useDocumentTitle('REPORT_SIGHTINGS');
 
   const { data: userData } = useGetMe();
-  const unprocessedAssetGroupId = get(userData, [
-    'unprocessed_asset_groups',
-    '0',
-  ]);
+  const unprocessedAssetGroups = get(userData, 'unprocessed_asset_groups', [])
+  const unprocessedBulks = unprocessedAssetGroups.filter(ag => get(ag, 'uploadType') !== 'form')
```
Comment (Atticus29) [inline src/pages/bulkImport/BulkImport.jsx:27]: Is this future-proof?

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -13,10 +13,9 @@ export default function Home() {
   const [crash, setCrash] = useState(false);
 
   const { data, loading, refresh } = useGetMe();
-  const unprocessedAssetGroupId = get(data, [
-    'unprocessed_asset_groups',
-    '0',
-  ]);
+  const unprocessedAssetGroups = get(data, 'unprocessed_asset_groups', [])
+  const unprocessedBulks = unprocessedAssetGroups.filter(ag => get(ag, 'uploadType') !== 'form')
```
Comment (Atticus29) [inline src/pages/home/Home.jsx:17]: Same here re: future-proof. Will "not form" always mean unProccessed?

**Review by Atticus29 — APPROVED** [review]
Some questions, but nothing deal-breaking.

File: src/pages/home/Home.jsx (hunk 2 above)
Comment (brmscheiner) [reply src/pages/home/Home.jsx:17]: AFAIK the answer is yes. We can always change it if necessary

**Pull request merged**